packages/shared_preferences/shared_preferences_linux/CHANGELOG.md

```diff
@@ -1,3 +1,8 @@
+## 2.1.0
+
+* Updated to make use of `dartPluginClass` only.
```

Comment:
I'm not sure what this means; it was already only using `dartPluginClass`. Isn't the change below the only actual change?